fe/fe-core/src/main/java/org/apache/doris/qe/ShowExecutor.java

```diff
@@ -449,6 +458,67 @@ private void handleShowMigrations() throws AnalysisException {
         resultSet = new ShowResultSet(showStmt.getMetaData(), rows);
     }
 
+    private void handleShowDbId() throws AnalysisException {
+        ShowDbIdStmt showStmt = (ShowDbIdStmt) stmt;
+        long dbId = showStmt.getDbId();
+        List<List<String>> rows = Lists.newArrayList();
+        Catalog catalog = ctx.getCatalog();
+        Database database = catalog.getDb(dbId);
+        if (database != null) {
+            List<String> row = new ArrayList<>();
+            row.add(database.getFullName());
+            rows.add(row);
+        }
+        resultSet = new ShowResultSet(showStmt.getMetaData(), rows);
+    }
+
+    private void handleShowTableId() throws AnalysisException {
+        ShowTableIdStmt showStmt = (ShowTableIdStmt) stmt;
+        long tableId = showStmt.getTableId();
+        List<List<String>> rows = Lists.newArrayList();
+        Catalog catalog = ctx.getCatalog();
+        List<Long> dbIds = catalog.getDbIds();
+        for (long dbId:dbIds) {
+            Database database = catalog.getDb(dbId);
+            Table table = database.getTable(tableId);
+            if (table != null) {
+                List<String> row = new ArrayList<>();
+                row.add(database.getFullName());
+                row.add(table.getName());
+                row.add(String.valueOf(database.getId()));
+                rows.add(row);
+            }
+        }
+        resultSet = new ShowResultSet(showStmt.getMetaData(), rows);
+    }
+
+    private void handleShowPartitionId() throws AnalysisException {
+        ShowPartitionIdStmt showStmt = (ShowPartitionIdStmt) stmt;
+        long partitionId = showStmt.getPartitionId();
+        List<List<String>> rows = Lists.newArrayList();
+        Catalog catalog = ctx.getCatalog();
+        List<Long> dbIds = catalog.getDbIds();
+        for (long dbId:dbIds) {
+            Database database = catalog.getDb(dbId);
+            List<Table> tables = database.getTables();
+            for (Table tbl:tables) {
```

Comment:
```suggestion
            for (Table tbl : tables) {
```
format

And we need tbl's readlock here.